build.gradle

```diff
@@ -98,6 +90,10 @@ task runSmokeTests(type: Exec, description: 'Runs smoke tests.') {
   commandLine '/usr/bin/yarn', '--silent', 'test:smoke'
 }
 
+task runFunctionalTests(type: Exec, description: 'Runs smoke tests.') {
```

Comment:
description and task name contradicts here

Reply:
copy and paste 🤦